### PR TITLE
Twindle web saurabh landing page

### PR DESCRIPTION
**Pull request by vasudeveloper001**
## Description
I changed some design of landing page with colors and others. 
- Feature section
- Header
- Image in Main
- Footer

Changed .Hbs template - added follow link for now.

## Attach Screenshot
screencapture-file-C-projects-github-open-source-twindle-twindle-web-index-html-2020-11-09-17_34_01

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

**Comment by proful** [comment]
Screen shots please @vasudeveloper001

**Comment by proful** [comment]
@vasudeveloper001 raise separate PR for web and cli projects.

@JoelVinayKumar can share his comments on new design

You need to fix merge conflicts properly. It wasn't done correctly

**Comment by vasudeveloper001** [comment]
Alright @proful 

It is showing me no conflict message but will wait for others to see what is wrong.

File: twindle-cli/env.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
+<<<<<<< HEAD:twindle-cli/twitter/utils/twitter-api-keys.js
+const { exit } = require("yargs");
+=======
 //  possibly include env related errors in a single location here
+>>>>>>> a9fd6f77e2de907238266312f792ea0050b70234:twindle-cli/env.js
```
Comment (arthurbuhl) [inline twindle-cli/env.js]: This is not correct. I think there were some merge conflicts which have not been resolved properly and the code was committed including these changes.

**Comment by vasudeveloper001** [comment]
Fixed that merge conflict but it will need some good review before merging just in case, I think.

**Comment by Akshay2996** [comment]
@vasudeveloper001 May I suggest to a site to have some inspiration of the background color.

Do visit this site ➡ [HappyHues](https://www.happyhues.co/) to get a good idea on the color theory🙂

**Comment by vasudeveloper001** [comment]
Thanks @Akshay2996

**Pull request closed without merging**